src/ray/stats/metric.h

```diff
@@ -112,6 +112,11 @@ class Metric {
         tag_keys_(tag_keys),
         measure_(nullptr) {}
 
+  Metric(const std::string &name,
```

Comment:
Can we keep only one constructor? Maybe we can remove the old one?

Reply:
It turns out that the constructor registered by Vector<TagKey> is used in many places. Mainly used in metric.pxi and jni.cc. It will be more troublesome to replace now.